Header header

uint16 x_input
uint16 y_input